packages/web-components/src/mixins/uswds-error-border.scss

```diff
@@ -8,6 +8,6 @@
 
 @media screen and (min-width: 1008px) {
     :host([error][uswds]:not([error=''])) {
-        margin-left: -1.4rem;
+        margin-left: -1.6rem;
```

Comment:
Error margin should be 16px

![Screenshot 2023-03-21 at 11 33 25 AM](https://user-images.githubusercontent.com/55560129/226657819-a37ab912-7713-4fd0-89cf-0bb5ff1a5b9b.png)